### DISCUSSION TITLE
quesion: minio server pools capacity

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#16429] [Q&A] [answered]
my minio cluster like follow:
![image](https://user-images.githubusercontent.com/27006237/213115885-268a002c-38e5-4ed0-9dc6-a8b54c14f298.png)
![image](https://user-images.githubusercontent.com/27006237/213117190-37076491-458b-4657-b0e4-7859df134987.png)

from above cluster, my minio have around 10GiB capacity, but when i use ```mc cp text1 dg/databend``` for copy text1 which size is 7GiB to minio , it throw follow error:
```
mc: <ERROR> Failed to copy `/opt/probe/text1`. Object does not exist
```

**Comment by klauspost** [comment, marked as answer]
MinIO reserves some overhead. You may just be toying around, but you don't gain any valuable information on such small installs.
